transport/wsn/src/test/scripts/org/kaazing/gateway/transport/wsn/auth/authorized.credentials.reponds.101.rpt

```diff
@@ -0,0 +1,16 @@
+#Normal gateway configuration with realm and authorization constraints

```

Comment:
Why are you adding any rpt files to gateway? Ideally all of the test scripts should be in the k3po specification project and the client side rpt's of the specification project should be used directly by the gateway ITs, i.e. I would expect to see only Java XxxIT classes in this pull request.